WordPress/gutenberg
# Components: fix `useCx` story to satisfy `exhaustive-deps` eslint rule

## What?

Ensure `useCx` stories satisfy the `exhaustive-deps` eslint rule.

## Why?

Part of the effort in https://github.com/WordPress/gutenberg/pull/41166 to apply `exhaustive-deps` to the Components package

## How?

The `useCx` utility includes a story with bad `useMemo` usages. In this case the missing dependency is expected and therefore we intend ignore it but it's on the wrong line.

Move the `eslint-disable-next-line` comment to the correct line.

## Testing Instructions

1. From your local Gutenberg directory, run `npx eslint --rule 'react-hooks/exhaustive-deps: warn' packages/components/src/utils/hooks/stories/use-cx.js`
3. Confirm that the linter returns no errors
